src/swarmus_pioneer/package.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<package format="2">
+    <name>swarmus_pioneer</name>
+    <version>0.0.0</version>
+    <description>The swarmus_pioneer package</description>
+
+    <!-- One maintainer tag required, multiple allowed, one person per tag -->
+    <!-- Example:  -->
+    <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+    <maintainer email="swarmus@usherbrooke.ca">SwarmUS</maintainer>
+
+
+    <!-- One license tag required, multiple allowed, one license per tag -->
+    <!-- Commonly used license strings: -->
+    <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+    <license>MIT</license>
+
+
+    <!-- Url tags are optional, but multiple are allowed, one per tag -->
+    <!-- Optional attribute type can be: website, bugtracker, or repository -->
+    <!-- Example: -->
+    <!-- <url type="website">http://wiki.ros.org/swarmus_ros_simulation</url> -->
+
+
+    <!-- Author tags are optional, multiple are allowed, one per tag -->
+    <!-- Authors do not have to be maintainers, but could be -->
+    <!-- Example: -->
+    <!-- <author email="jane.doe@example.com">Jane Doe</author> -->
+
+
+    <!-- The *depend tags are used to specify dependencies -->
+    <!-- Dependencies can be catkin packages or system dependencies -->
+    <!-- Examples: -->
+    <!-- Use depend as a shortcut for packages that are both build and exec dependencies -->
+    <!--   <depend>roscpp</depend> -->
+    <!--   Note that this is equivalent to the following: -->
+    <!--   <build_depend>roscpp</build_depend> -->
+    <!--   <exec_depend>roscpp</exec_depend> -->
+    <!-- Use build_depend for packages you need at compile time: -->
+    <!--   <build_depend>message_generation</build_depend> -->
+    <!-- Use build_export_depend for packages you need in order to build against this package: -->
+    <build_export_depend>message_generation</build_export_depend>
+    <!-- Use buildtool_depend for build tool packages: -->
+    <!--   <buildtool_depend>catkin</buildtool_depend> -->
+    <!-- Use exec_depend for packages you need at runtime: -->
+    <!--   <exec_depend>message_runtime</exec_depend> -->
+    <!-- Use test_depend for packages you need only for testing: -->
+    <!--   <test_depend>gtest</test_depend> -->
+    <!-- Use doc_depend for packages you need only for building documentation: -->
+    <!--   <doc_depend>doxygen</doc_depend> -->
+    <buildtool_depend>catkin</buildtool_depend>
+
+    <build_depend>controller_manager</build_depend>
+    <build_depend>joint_state_controller</build_depend>
+    <build_depend>robot_state_publisher</build_depend>
+    <build_depend>roscpp</build_depend>
+    <build_depend>std_msgs</build_depend>
+    <build_depend>geometry_msgs</build_depend>
+    <build_depend>tf</build_depend>
+    <build_depend>message_generation</build_depend>
+    <build_depend>kinect</build_depend>
+    <build_depend>openni_camera</build_depend>
+    <build_depend>openni_tracker</build_depend>
+
```

Comment:
Good point. Since their dependency only shows up at the launch file, it should be an exec_depend, unless we need to create node in the future that will use the messages generated by those packages

Reply:
Actually I just realize that the MPU6050 will not be used since there is an IMU in the two realsense, so we can actually remove its node and everything related to it. Sorry for that I should have told you before